src/components/Loader/Loader.jsx

```diff
@@ -1,23 +1,36 @@
 // @flow
 import React from 'react';
+import cn from 'classnames';
 import style from './style.scss';
 import { getDataAttrs } from '../../dataUtils';
 import type { Data } from '../../types';
 
 type Props = {|
+  className?: Object,
+  dotColor?: string,
+  dotClassName?: Object,
   style?: Object,
   dotStyle?: ?Object,
   data?: Data,
 |};
 
 class Loader extends React.Component<Props> {
   render() {
+    const dotStyle = this.props.dotStyle || {};
```

Comment:
make this a default prop?

Reply:
WDYT about https://github.com/wework/plasma/pull/243/files#diff-529cbf4c11cef5ef490162beb870bc08R19 ?

Reply:
that's fine too, I was just thinking to follow a more idiomatic-reacty way to define a default value for a prop

Reply:
my bad, I kept defining `dotStyle?: Object` when I tried the `defaultProps`. I forgot that `dotStyle: Object` is enough.